portfolio/src/main/webapp/script.js

```diff
@@ -53,3 +53,27 @@ function getHelloNzingaUsingAsyncAwait() {
       });
     });
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Animal');
+  data.addColumn('number', 'Count');
+        data.addRows([
+          ['Brazil!', 10],
+          ['Mexico!', 5],
+          ['Alaska!', 15]
+        ]);
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation): Each time a new block or block-like construct is opened, the indent increases by two spaces.

[Style](https://google.github.io/styleguide/jsguide.html#formatting-indent): When line-wrapping, each line after the first (each continuation line) is indented at least +4 from the original line, unless it falls under the rules of block indentation.

It should look something like this:
```suggestion
  data.addRows([
      ['Brazil!', 10],
      ['Mexico!', 5],
      ['Alaska!', 15]
  ]);
```

Again, feel free to address this in PR #12 instead so that you don't have to deal with merge conflicts here.